cmd/plume/containerlinux.go

```diff
@@ -141,6 +141,19 @@ var (
 			),
 			AWS: newAWSSpec(),
 		},
+		"lts": channelSpec{
+			BaseURL:      "gs://flatcar-jenkins-private/lts/boards",
+			Boards:       []string{"amd64-usr"},
+			Destinations: []storageSpec{},
+			GCE:          gceSpec{},
+			Azure: newAzureSpec(
+				azureEnvironments,
+				"publish",
+				"Flatcar LTS",
```

Comment:
There is another category field that is added for Pro/Regular

Reply:
Thanks for catching this, rebased and updated